common/rpc_service.go

```diff
@@ -24,14 +24,9 @@ import (
 	"sync"
 	"unicode"
 	"unicode/utf8"
-)
 
-import (
-	perrors "github.com/pkg/errors"
-)
-
-import (
 	"github.com/apache/dubbo-go/common/logger"
+	perrors "github.com/pkg/errors"
```

Comment:
hey, guy, pls do not change our import code format. recover it pls.

Reply:
@AlexStocks Hi, this is `vim-go` 's behavior.  I think it 'll change  `import order` sorted by alphabet. Can you accept this?

Reply:
> @AlexStocks Hi, this is `vim-go` 's behavior. I think it 'll change `import order` sorted by alphabet. Can you accept this?

no.  we dubbo-go all code files follow the same import code style.

Reply:
@AlexStocks Sure, I will keep it as the same. ps, Could u share the  `import code style`?

Reply:
refer to https://github.com/apache/dubbo-go/blob/develop/contributing.md